mappings/net/minecraft/loot/function/SetEnchantmentsLootFunction.mapping

```diff
@@ -9,7 +13,9 @@ CLASS net/minecraft/unmapped/C_bpyfibfg net/minecraft/loot/function/SetEnchantme
 		ARG 2 level
 	METHOD m_exjzhadm (Lnet/minecraft/unmapped/C_iajmfyig;)Ljava/util/stream/Stream;
 		ARG 0 numberProvider
-	METHOD m_grdiinyw (Lit/unimi/dsi/fastutil/objects/Object2IntMap;Lnet/minecraft/unmapped/C_iakykpgh;Lnet/minecraft/unmapped/C_jxtrubuh;Lnet/minecraft/unmapped/C_iajmfyig;)V
+	METHOD m_grdiinyw addEnchantmentToMap (Lit/unimi/dsi/fastutil/objects/Object2IntMap;Lnet/minecraft/unmapped/C_iakykpgh;Lnet/minecraft/unmapped/C_jxtrubuh;Lnet/minecraft/unmapped/C_iajmfyig;)V
```

Comment:
This appears to be synthetic (lambda I assume?), so iiuc it shouldn't be mapped?
```java
    private static /* synthetic */ void addEnchantmentToMap(Object2IntMap map, LootContext context, Enchantment enchantment, LootNumberProvider numberProvider) {
        map.put((Object)enchantment, numberProvider.nextInt(context));
    }
```

Reply:
It is a lambda, why shouldn't it be mapped?

Reply:
Because lambdas don't have names?
I think I've seen another comment saying a mapping for a lambda should be removed, but I'm not sure if it's actually an issue. I'll ask in in discord, I may be way off on this.

Reply:
Apparently it doesn't matter too much if lambdas are mapped or not.